### ISSUE TITLE
Ensure that a clean vertical project package gives no build warnings/errors

### ISSUE THREAD

**Issue by mvuorre** [CrumpLab/vertical#46]
Currently you get

```
── R CMD check results ────────────────────────────────── test87 0.0.0.9000 ────
Duration: 7.2s

> checking DESCRIPTION meta-information ... WARNING
  Invalid license file pointers: LICENSE

> checking files in ‘vignettes’ ... WARNING
  Files in the 'vignettes' directory but no files in 'inst/doc':
    ‘som.Rmd’
  Package has no Sweave vignette sources and no VignetteBuilder field.

> checking top-level files ... NOTE
  Non-standard file/directory found at top level:
    ‘_pkgdown.yml’

0 errors ✓ | 2 warnings x | 1 note x
Error: R CMD check found WARNINGs
Execution halted

Exited with status 1.
```

**Comment by CrumpLab** [comment]
What are your thoughts on addressing the license warning?

I'm guessing we would eliminate the LICENSE warning by including a license file...which would be easy enough to do, however do we want to be that prescriptive?

I think I can get rid of the other two warnings, let me check

**Comment by CrumpLab** [comment]
Adding this during initialization, should take care of the the `_pkgdown.yml` warning

```
  usethis::use_template(template = "_pkgdown.yml",
                        ignore = TRUE,
                        package = "vertical")
```

**Comment by CrumpLab** [comment]
All fixed in c723face9022fe6a48f0190cce75e1ff0288ea9e

MIT license is added automatically
